### ISSUE TITLE
Support scroll option based on next router.push in setLanguage

### ISSUE THREAD

**Issue by zyl-edison** [vinissimus/next-translate#674]
After we use [setLanguage](https://github.com/vinissimus/next-translate/blob/master/src/setLanguage.tsx) in the middle position of the page, it always scroll to the top because the scroll option in router.push is true by default. see ref below

Ref: https://nextjs.org/docs/api-reference/next/router#routerpush

Since we do not want to scroll to the top of the page, maybe support something like

```
export default async function setLanguage(locale: string, scroll = true): Promise<boolean> {
  return await Router.push(Router.pathname, Router.asPath, {
    locale,
    scroll
  })
}
```

**Comment by alexllao** [comment]
I have the same problem.

**Comment by aralroca** [comment]
Thank you for reminding me. I have made the change, available now in the [1.3.0-canary.5](https://github.com/vinissimus/next-translate/releases/tag/1.3.0-canary.5) prerelease.

**Comment by alexllao** [comment]
Thnks a lot!
